pkg/cmd/dev/ui.go

```diff
@@ -248,3 +247,205 @@ Replaces 'make ui-lint'.`,
 
 	return lintCmd
 }
+
+// arrangeFilesForTestWatchers moves files from Bazel's build output directory into the locations they'd be found during
+// a non-Bazel build, so that test watchers can successfully operate outside of the bazel sandbox.
```

Comment:
Some additional context about how you tried to get it work without the copying but you couldn't get watcher to follow the symlinks would probably be helpful. Would at least be a hint to anyone else investigating this area in the future.

Also, nit, these comment lines look a little long. 80 characters?

Reply:
Good point!  I'll clarify things.

`gofmt -s -w` didn't complain about the width, oddly enough.  I'll re-wrap it anyway ✂️

Reply:
Pretty sure `gofmt` won't mess with comment blocks in case they contain preformatted stuff like diagrams or tables.